backend/src/main/java/com/google/sps/datastore/ParticipantDatastore.java

```diff
@@ -37,7 +37,7 @@
 import javax.annotation.Nullable;
 
 /** Separates datastore method calls involving Participant type from caller */
-public final class ParticipantDatastore {
+public class ParticipantDatastore {
```

Comment:
Quick question, this class is no longer final because ?

Reply:
Great question, this class is no longer final because it needed to be mocked in the tests and final classes cannot be mocked.